app/commands/github/issue/search.rb

```diff
@@ -0,0 +1,75 @@
+module Github
+  class Issue::Search
+    include Mandate
+
+    # Use class method rather than constant for
+    # easier stubbing during testing
+    def self.requests_per_page
+      20
+    end
+
+    def initialize(action: nil, knowledge: nil, module_: nil, size: nil, type: nil, repo: nil, order: nil, page: 1)
+      @action = action
+      @knowledge = knowledge
+      @module_ = module_
+      @size = size
+      @type = type
+      @repo = repo
+      @order = order
+      @page = page
+    end
+
+    def call
+      @issues = Github::Issue
+
+      filter_repo!
+      filter_unclaimed!
+      filter_action!
+      filter_knowledge!
+      filter_module!
+      filter_size!
+      filter_type!
+
+      sort!
+      paginate!
+    end
+
+    private
+    attr_reader :action, :knowledge, :module_, :size, :type, :repo, :order, :page, :issues
+
+    def filter_repo!
+      return if repo.blank?
+
+      @issues = @issues.where(repo: repo)
+    end
+
+    def filter_unclaimed!
+      @issues = @issues.without_label('x:status/claimed')
+    end
+
+    %w[action knowledge module_ size type].each do |label|
+      normalized_label = label.delete('_')
```

Comment:
This is needed for the `_` suffix on the `module_` value. This can be removed if we decide on another name.